### DISCUSSION TITLE
Reading shipping services via API

### DISCUSSION THREAD

**Discussion by vettloffah** [swellstores/community#143] [Help] [answered]
Is there an endpoint get the "Shipping Services" settings via API?

We are writing our own shipping calculation logic and would like to continue to manage the base rates via the dashboard.

**Comment by vettloffah** [comment, marked as answer]
Ah I found it in the `/settings` endpoint

**Reply by ericingram** [reply]
You got it!

Generally, every bit of data in Swell is available through an API endpoint. We need to improve the docs to make that more obvious.
